includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -2217,6 +2217,10 @@ protected function prepare_payment_information_from_request( WC_Order $order ) {
 			'return_url'                    => $this->get_return_url_for_redirect( $order, $save_payment_method_to_store ),
 			'use_stripe_sdk'                => 'true', // We want to use the SDK to handle next actions via the client payment elements. See https://docs.stripe.com/api/setup_intents/create#create_setup_intent-use_stripe_sdk
 			'has_subscription'              => $this->has_subscription( $order->get_id() ),
+			'payment_method'                => '',
+			'payment_method_details'        => [],
+			'payment_type'                  => 'single', // single | recurring.
+			'capture_method'                => $capture_method,
```

Comment:
@wjrosa Can we exclude the default capture method at this level? Amazon Pay doesn't like it and complains (when capture method is manual), because it needs to be under payment_method_options?

Reply:
Oh OK! I drafted this PR to revert the setting for this specific parameter here https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4019